Resources/NetHook2/NetHook2/crypto.cpp

```diff
@@ -13,41 +14,53 @@
 SymmetricEncryptChosenIVFn Encrypt_Orig = nullptr;
 PchMsgNameFromEMsgFn PchMsgNameFromEMsg = nullptr;
 
+#ifndef __x86_64__

```

Comment:
Would prefer to leave this in, just checking for `sizeof(MsgInfo_t) == 24` on `__x86_64`. The other asserts should pass without issue on x64.